NEWS.md

```diff
@@ -1,3 +1,16 @@
+# devel
+
+## New features
+
+* The `rhub_check_list` class gained a `get_group()` and a `get_check()` methods returning a `rhub_check` object from the list, by group_id or check_id.
```

Comment:
I think the classes won't be exported? In this case we should not write news about them.

Reply:
I disagree, the new methods are an important update to what you can do to the output of list functions?

Reply:
Sure, I meant keeping this bullet, but omitting the class names somehow.

Reply:
I.e. just write about the values returned by `check_*()` functions?

Reply:
Ah right will fix that!

Reply:
is the new phrasing better?